_posts/en/newsletters/2023-10-18-newsletter.md

```diff
@@ -0,0 +1,154 @@
+    order to spend bitcoins associated with that script.  For example, a
+    script containing a public key that can only be satisfied if the
+    corresponding private key creates a signature committing to a
+    spending transaction.  Scripts must be written in Bitcoin's language
+    (called _Script_) in order to be enforced by consensus, but Script
+    is deliberately limited in its flexibility.
+
+    Linus's paper gets around some of those limits.  If Alice trusts Bob
+    to take action if a program was run incorrectly, but does not want
+    to trust him with anything else, she can pay funds to a taproot tree
+    that will allow Bob to claim the funds if he demonstrates that Alice
+    failed to run an arbitrary program correctly.  If Alice does run the
+    program correctly, then she can spend the funds even if Bob attempts
+    to stop her.
+
+    To use an arbitrary program, it must be broken down into a very
+    basic primitive (a NAND gate) and a commitment must be made for each
+    gate.  This requires the offchain exchange of a very large amount of
+    data, potentially multiple gigabytes for even a fairly basic
+    arbitrary program---but Alice and Bob only need a single onchain
+    transaction in the case that Bob agrees that Alice ran the program
+    correctly.  In the case that Bob disagrees, he should be able to
+    demonstrate Alice's failure within a relatively small number of
+    onchain transactions.  If the setup was performed in a payment
+    channel between Alice and Bob, multiple programs could be run in
+    both parallel and in sequence with no onchain footprint except
+    channel setup and either a mutual close or a force close where Bob
+    attempts to demonstrate that Alice failed to follow the arbitrary
+    program logic correctly.
+
+    BitVM can be applied trustlessly in cases where Alice and Bob are
+    natural adversaries, such as where they pay funds to an output that
+    will be paid to whichever one of them wins in a game of chess.  They
+    can then use an two (almost identical) arbitrary programs, each of
+    which takes the same arbitrary set of chess moves.  One program will
+    return true if Alice won and one will return true if Bob won.  One
+    party will then publish onchain the transaction that claims their
+    program evaluates to true (that they won); the other party will
+    either accept that claim (conceding the loss of funds) or will
+    demonstrate the fraud (receiving the funds if successful).  In cases
+    where Alice and Bob would not naturally be adversaries, Alice may be
+    able to incentivize him to verify correct computation, such by
+    offering her funds to Bob if he can prove she failed to compute
+    correctly.
+
+    The idea received a significant amount of discussion on the mailing
+    list as well as on Twitter and various Bitcoin-focused podcasts.  We
+    expect significant continued discussion in the coming weeks and
```

Comment:
```suggestion
    expect substantial continued discussion in the coming weeks and
```
(Nit, just so we're not using "significant" twice close together. Or drop this word.)